.github/workflows/ci.yaml

```diff
@@ -741,14 +741,14 @@ jobs:
         run: |
           . venv/bin/activate
           python --version
-          pylint --ignore-missing-annotations=y homeassistant
+          pylint homeassistant
```

Comment:
Please don't change other files than the ones related to the integration you're adding to changing.